aqt/installer.py

```
@@ -836,8 +836,10 @@ def run_install_qt_commercial(self, args: InstallArgParser, print_version: Optio
 
             commercial_installer.install()
             Settings.qt_installer_cleanup()
+        except AqtException:
+            raise
         except Exception as e:
-            self.logger.error(f"Error installing official installer: {str(e)}")
+            raise AqtException(f"Error installing official installer: {str(e)}") from e
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)